aiosmtpd/testing/helpers.py

```diff
@@ -26,59 +26,55 @@ def reset_connection(client):
     # };
     #
     # Is this correct for Windows/Cygwin and macOS?
-    struct_format = 'hh' if sys.platform == 'win32' else 'ii'
+    struct_format = "hh" if sys.platform == "win32" else "ii"
     l_onoff = 1
     l_linger = 0
     client.sock.setsockopt(
         socket.SOL_SOCKET,
         socket.SO_LINGER,
-        struct.pack(struct_format, l_onoff, l_linger))
+        struct.pack(struct_format, l_onoff, l_linger),
+    )
     client.close()
 
 
-# For integration with flufl.testing.
-
-def setup(testobj):
-    testobj.globs['resources'] = ExitStack()
-
-
-def teardown(testobj):
-    testobj.globs['resources'].close()
-
-
-def make_debug_loop():
-    loop = asyncio.get_event_loop()
-    loop.set_debug(True)
-    return loop
```

Comment:
These are required for flufl.testing only? Anyways, not used by any test case, so I'm removing them.